app/controllers/hello_controller.rb

```diff
@@ -0,0 +1,6 @@
+class HelloController < ApplicationController
+
+  def index
+  end
+
+end
```

Comment:
As I remember you have the `dummy_feature_spec.rb`. It doesn't dummy since that moment, right? Let's rename it in order to be more appropriate to our greeting page.

Reply:
Well, I could argue here. I would say that whole controller is rather a dummy then feature_spec is no dummy. Anyway, whole spec and controller will be removed when we will proceed to real pages and controllers.

But ok, for now, I will do the respective changes